### PR TITLE
Add lint command and contributing instructions for linting

### PR DESCRIPTION
**Pull request by sonkkeli**
Also lint existing mark-up files, which haven't been touched so far.

### CODE REVIEW

File: js/bundle/package.json [hunk 1]
```diff
@@ -14,7 +14,8 @@
     "build": "npm run build:esm && npm run build:cjs",
     "build:esm": "tsc",
     "build:cjs": "esbuild --bundle --format=cjs --external:cborg --outfile=lib/wbn.cjs src/wbn.ts",
-    "test": "jasmine tests/*.js tests/*.cjs"
+    "test": "jasmine tests/*.js tests/*.cjs",
+    "lint": "npx prettier --write **/*.{js,md} --config ./package.json"
```
Comment (cmfcmf) [inline js/bundle/package.json:18]: Should this also include `ts`?

File: js/sign/package.json [hunk 2]
```diff
@@ -14,7 +14,8 @@
     "build": "npm run build:esm && npm run build:cjs",
     "build:esm": "tsc",
     "build:cjs": "esbuild --bundle --format=cjs --external:cborg --outfile=lib/wbn-sign.cjs src/wbn-sign.ts --platform=node",
-    "test": "jasmine tests/*.js tests/*.cjs"
+    "test": "jasmine tests/*.js tests/*.cjs",
+    "lint": "npx prettier --write **/*.{js,md} --config ./package.json"
```
Comment (cmfcmf) [inline js/sign/package.json:18]: Should this also include `ts`?

**Review by cmfcmf — APPROVED** [review]
LGTM % question regarding linting of TypeScript files.

File: js/bundle/package.json (hunk 1 above)
Comment (sonkkeli) [reply js/bundle/package.json]: Yes, fixed!

**Review by irori — APPROVED** [review]

**Pull request merged**